saleor/graphql/shop/resolvers.py

```diff
@@ -0,0 +1,35 @@
+from ...core.taxes import display_gross_prices
+from ...plugins.manager import get_plugins_manager
+from ...shipping.models import ShippingMethod
+from ..channel import ChannelContext
+
+
+def resolve_available_shipping_methods(channel_slug: str, address):
+    available = ShippingMethod.objects.filter(
+        channel_listings__channel__slug=channel_slug
+    )
+    if address and address.country:
+        available = available.filter(
+            shipping_zone__countries__contains=address.country,
+        )
+
+    if available is None:
+        return []
+    manager = get_plugins_manager()
+    display_gross = display_gross_prices()
```

Comment:
This information comes from `SiteSettings` object and I think it should be already prefetched as well in the info object: `info.context.site.settings.display_gross_prices`. This function doesn't seem too useful, I think we will drop it later during performance fixes.